### ISSUE TITLE
Increase tap hit target height of progress bar of new player to 1/5 of height landscape

### ISSUE THREAD

**Issue by paaspaas00** [TeamNewPipe/NewPipe#12750]
### Checklist

- [x] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [x] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [x] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [x] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [x] This issue contains only one feature request.
- [x] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

The new player's progress bar requires too much precision when trying to tap the progress bar to seek.

It should be more forgiving, and allow you to tap on any place of the lower portion of the screen.
About 1/5 of landscape screen height (meaning all the lower portion of the screen in landscape mode with rectangle height 1/5 screen height) should be fine I think.

Currently if you're not precisely tapping the progress bar, it triggers other events gestures like app switch to another app.

### Why do you want this feature?

It makes easier to seek and improves usability of new player.
